src/com/lftechnology/java/training/srijan/basics/GuessingNumbers.java

```diff
@@ -0,0 +1,47 @@
+package com.lftechnology.java.training.srijan.basics;
+
+
+import java.security.SecureRandom;
+import java.util.Scanner;
+import java.util.logging.Logger;
+/**
+ * Write a Guessing a number game application: 
+ * application will generate a random no between 1 and 20. 
+ * user guess the value by entering from the console
+ * Check if the guess value matched with random number or not, 
+ * if matched show the guessing value, random value and the no. of attempts.
+ */
+public class GuessingNumbers {
+	private int numberOfAttempt;
+	private static final Logger LOGGER = Logger.getLogger(GuessingNumbers.class.getName());
+	
+	public GuessingNumbers(){
+		numberOfAttempt = 0;
+	}
+	
+	public static void main(String[] args){
+		Scanner input = null;
+		SecureRandom randomNumber = new SecureRandom();
+		GuessingNumbers guessNumber = new GuessingNumbers();
```

Comment:
why are you making object  of GuessingNumbers inside GuessingNumber method :question: 
The main method should be in different class.